package.json

```diff
@@ -9,16 +9,13 @@
   "packageManager": "npm@10.7.0",
   "scripts": {
     "elm-format": "elm-format --validate ast-codec init-templates new-package parseElm review template",
-    "test": "npm-run-all --print-name --silent elm-tests elm-format tsc eslint-check prettier-check test-run",
```

Comment:
Do you know if it's possible to keep a test script so that I can just run `npm test` ?